vhd/vhd.go

```diff
@@ -1,3 +1,4 @@
+//go:build windows
 // +build windows
```

Comment:
Do we need both tags?

Reply:
gofmt will automatically apply the go 1.17 syntax of build tags to anything if you're on 1.17. Our go.mod here lists 1.12 as the earliest go version that you could build this project with, so we'd have to have both for the time being. However, what I've seen from perusing around yee olde github (https://cs.github.com/?q=%22%2F%2Fgo%3Abuild%22) is the overwhelming majority of folks even if they updated their go directive in their go.mod to be 1.17 is they've kept both syntaxes.